src/util.ts

```diff
@@ -2,7 +2,13 @@ export type DispatchOptions = { target: EventTarget, cancelable: boolean, detail
 
 export function dispatch(eventName: string, { target, cancelable, detail }: Partial<DispatchOptions> = {}) {
   const event = new CustomEvent(eventName, { cancelable, bubbles: true, detail })
-  void (target || document.documentElement).dispatchEvent(event)
+
+  if (target && document.body.contains(target as Element)) {
+    void target.dispatchEvent(event);
```

Comment:
What is the significance of the `void` here? Could it be omitted?

Reply:
Good question. Left that as it was, but it did strike me as odd.

Reply:
Couldn't find a purpose. Stripped it 👍